packages/kit/src/packaging/index.js

```diff
@@ -21,8 +21,8 @@ export async function make_package(config, cwd = process.cwd()) {
 
 	const files_filter = create_filter(config.kit.package.files);
 	const exports_filter = create_filter({
-		...config.kit.package.exports,
-		exclude: [...config.kit.package.exports.exclude, '*.d.ts']
+		include: config.kit.package.exports.include,
```

Comment:
Why does this need to be set explicitely? Isn't it part of the spread already?

Reply:
Always found myself taking a while to read and understand when I get to this part. Explicitly setting the include option makes it clearer as it now has the same indentation, and avoids spreading and immediately overriding the exclude values.

Reply:
Makes sense 👍